### PR TITLE
[ADD] Create Odoo Docker Core

### PR DESCRIPTION
**Pull request by lasley**
Take a look at the [Readme](https://github.com/LasLabs/docker-odoo/blob/feature/alpine/README.md) for all the details. It was rather dense to paste in here as a comment.

* Build and install Odoo on Alpine
* Support all Odoo versions in one file w/ env vars
* Automagically determine addon directories using OCA/MQT/get_addons
* Automagically install dependencies from mounted volume using `oca_dependencies.txt` and `requirements.txt` in root of dir

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,31 +1,170 @@
...
 
-* Go to ...
 
 Known Issues / Roadmap
 ======================
 
-* ...
+* Create a Python base.
+* Create an Python-Wkhtmltox-XVFB base.
+* Add oca_clone_everything for default modules.
```
Comment (lasley) [inline README.md]: This is done

File: README.md [hunk 2]
```diff
@@ -1,31 +1,188 @@
...
+[![Build Status](https://travis-ci.org/LasLabs/docker-odoo.svg?branch=master)](https://travis-ci.org/LasLabs/docker-odoo)
+
+Alpine Odoo
+===========
+
+This provides an Odoo instance for Docker using Alpine Linux.
+
+It aims to operate in the exact same way as the official Odoo Docker image,
+although there are also some additional features that are helpful for developers
+intending to use OCA modules.
```
Comment (tedsalmon) [inline README.md:11]:
```
s/to/the
```
Unless I'm reading this wrong :)

File: .travis.yml [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+  - docker pull kiasaki/alpine-postgres
+  - docker run -d -e POSTGRES_USER=odoo -e POSTGRES_PASSWORD=odoo --name db kiasaki/alpine-postgres
+
+install:
+  - docker build -t $TRAVIS_BUILD_NUMBER --build-arg ODOO_VERSION=$ODOO_VERSION $TRAVIS_BUILD_DIR/
+
+script:
+  - docker run -d -p 8069:8069 --name $TRAVIS_BUILD_NUMBER --link db:db -t $TRAVIS_BUILD_NUMBER
+  - sleep 10
+  - curl --fail http://localhost:8069/web/database/manager
```
Comment (tedsalmon) [inline .travis.yml:23]: Perhaps we should:
```
curl -s --fail http://localhost:8069/web/database/manager > /dev/null
```
The Travis is a little messy due to the HTML output.

**Review by tedsalmon — APPROVED** [review]
👍  LGTM minus a couple minor details

File: .travis.yml (hunk 3 above)
Comment (lasley) [reply .travis.yml:23]: I considered that, but typically Travis  is a log of everything. What if it fails, for example?

File: README.md (hunk 2 above)
Comment (lasley) [reply README.md:11]: I'm pretty sure to is correct here - `developers intending the use OCA modules` sounds weird IMO

Comment (tedsalmon) [reply README.md:11]: Yeah, I was reading it wrong. No idea why I thought the sounded better... (derp)

File: .travis.yml (hunk 3 above)
Comment (tedsalmon) [reply .travis.yml:23]: Yeah, I guess we'd like to see the failure in that scenario. Cool, makes sense.

**Pull request merged**